homework/04LessonFour.js

```diff
@@ -77,7 +91,7 @@ var nestedObjects = {
   }
 };
 // update the starring property to have the value 'Tim Robbins'
-
+nestedObjects.users['bob roberts'].starring='Tim Robbins';
```

Comment:
Nice use of bracket and dot notation!

![dancing](https://media.giphy.com/media/l46CdUTy21h8wwrAs/giphy.gif)